pkg/storage/trb_request_system_test.go

```diff
@@ -0,0 +1,128 @@
+package storage
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/sqlutils"
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+)
+
+func (s *StoreTestSuite) TestLinkTRBRequestSystems() {
+	ctx := context.Background()
+
+	const (
+		system1 = "1"
+		system2 = "2"
+		system3 = "3"
+		system4 = "4"
+	)
+
+	var createdIDs []uuid.UUID
+
+	s.Run("sets systems on a trb request", func() {
+		// create three trb requests
+		_, err := sqlutils.WithTransaction[any](s.db, func(tx *sqlx.Tx) (*any, error) {
+			for i := 0; i < 3; i++ {
+				trbRequest := models.NewTRBRequest(testhelpers.RandomEUAIDNull().String)
+				trbRequest.Type = models.TRBTBrainstorm
+				trbRequest.State = models.TRBRequestStateOpen
+				created, err := s.store.CreateTRBRequest(ctx, tx, trbRequest)
+				s.NoError(err)
+				createdIDs = append(createdIDs, created.ID)
+			}
+			return nil, nil
+		})
+		s.NoError(err)
+
+		// insert systems for this created system intake
```

Comment:
nit: `system intake` -> `trb request`

Reply:
Addressed in [ca20523](https://github.com/CMSgov/easi-app/pull/2445/commits/ca20523b290c8a45960d581688769f1feb3c66a8)